discord/bot.py

```diff
@@ -25,10 +25,11 @@
 import asyncio
 import inspect
 
+from .ext.commands._types import _BaseCommand
 from .client import *
 
 
-class ApplicationCommand:
+class ApplicationCommand(_BaseCommand):
```

Comment:
Not sure if this is intentional but this was reverted in a previous commit, perhaps you forgot to pull?

Reply:
Strange, I had pulled it